src/components/data-management/ProjectCard.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card, Descriptions,
+} from 'antd';
+import { blue } from '@ant-design/colors';
+import { useSelector } from 'react-redux';
+import EditableField from '../EditableField';
+import PrettyTime from '../PrettyTime';
+
+const { Item } = Descriptions;
+
+const activeProjectStyle = {
+  backgroundColor: blue[0],
+  cursor: 'pointer',
+  border: `2px solid ${blue.primary}`,
+};
+
+const ProjectCard = (props) => {
+  const {
+    uuid, isActive, onClick, onSubmit, onDelete, validationFn,
+  } = props;
+
+  const project = useSelector((state) => state.projects[uuid]);
+
+  return (
+    <Card
+      data-test-class='project-card'
+      key={uuid}
+      type='primary'
+      style={isActive ? activeProjectStyle : { cursor: 'pointer' }}
```

Comment:
It will make the code cleaner I feel if you also have a variable for the style of the inactive project and use it here similarly to `activeProjectStyle`:

```
const inactiveProjectStyle = {
    cursor: 'pointer'
}

....

style={isActive ? activeProjectStyle : inactiveProjectStyle}
```